fix: crawler-link-extraction-order.test.ts の CrawlConfig に version プロパティ欠落

## 問題

mainブランチのCI（Type Check）が失敗。

```
tests/unit/crawler-link-extraction-order.test.ts(50,3): error TS2741: 
Property 'version' is missing in type '...' but required in type 'CrawlConfig'.
```

## 原因

PR #718（リンク抽出順序のリグレッションテスト追加）が PR #734（CrawlConfig version修正）の**後に**マージされたが、`version` プロパティが含まれていなかった。

## 修正箇所

`link-crawler/tests/unit/crawler-link-extraction-order.test.ts` の `baseConfig` オブジェクト（50行目付近）に `version: "1.0.0"` を追加する。

```diff
  keepSession: false,
  respectRobots: false,
+ version: "1.0.0",
};
```

## 関連

- PR #718: リグレッションテスト追加
- PR #734 / Issue #729: CrawlConfig version修正（他テストファイルは修正済み）

--- a/link-crawler/tests/unit/crawler-link-extraction-order.test.ts
+++ b/link-crawler/tests/unit/crawler-link-extraction-order.test.ts
@@ -65,6 +65,7 @@ describe("Crawler - link extraction order", () => {
 			chunks: false,
 			keepSession: false,
 			respectRobots: false,
+			version: "1.0.0",
 		};
 	});
 

